ssh/README.md

```diff
@@ -25,6 +25,9 @@ Follow these steps to get the add-on installed on your system:
 
 ## How to use
 
+You need enable the port for external access. You can just enter 22 as value or any other
```

Comment:
IMHO This doc change isn't helpful to a large number of people.   It just doesn't make sense unless the reader knows exactly what you are talking about. This was a breaking change. See new issue #1125.

Reply:
You've opened up an issue, not sure why you would create a comment here as well.

Please don't dig up closed PR's. Thanks.